BUILDING.txt

```diff
@@ -36,12 +39,18 @@ Build Requirements (Unix)
 =========================
 
 -- Non-Mac platforms:
-   * X11 development kit
+  Development kit for the following:
+   * X11
+   * Xtst
+   * Xfixes
+   * Xrandr
+   * Xdamage
```

Comment:
Being this explicit worries me. This list tends to change, so I doubt we'll pay attention to updating this document. The existing message was meant to mean "all standard X11 libraries".

And some are optional as well, right?

Reply:
I agree that using "all standard X11 libraries" would be a neat way to express this, but it is not very helpful when unfamiliar with X11. If not mentioning explicit packages, could we mention higher level themes or similar to give a better starting point?

I have not looked into if the packages are optional or not.

Reply:
I decided to go along the line of "all standard X11 libraries". Since the additional packages likely were a quite incomplete list I didn't include those.